docs/_tutorials/ldap.md

```diff
@@ -0,0 +1,179 @@
+* You are familiar with [Cloud Foundry](https://www.cloudfoundry.org/){:target="_blank"}.
+* You have access to a running Pivotal Cloud Foundry environment.
+* Solace Messaging for PCF has been installed in your Pivotal Cloud Foundry environment.
+* You have completed the Java app tutorial.
+* You are using OpenLDAP. Some things may be different if you are using a different LDAP implementation.
+
+## Files
+
+This section only concerns you if you are setting up your own LDAP server.
+
+If you already have a configured LDAP server then you can skip this section.
+
+#### memberOf.ldif
+
+The memberOf overlay is mainly for convenience.
+
+With the memberOf overlay whenever the list of members in a group is updated, that member automatically gets updated with the correct memberOf attribute as well. This means only updating one place instead of two. 
+
+Without the memberOf overlay, a memberOf attribute for every user has to be updated manually with the correct groups.
+
+```
+dn: cn=modulecn=config
+objectClass: olcModuleList
+cn: module
+olcModulePath: /usr/lib64/openldap
+olcModuleLoad: memberof
+
+dn: olcOverlay=memberofolcDatabase={2}bdbcn=config
+objectClass: olcMemberOf
+objectClass: olcOverlayConfig
+objectClass: olcConfig
+objectClass: top
+olcOverlay: memberof
+olcMemberOfDangling: ignore
+olcMemberOfRefInt: TRUE
+olcMemberOfGroupOC: groupOfNames
+olcMemberOfMemberAD: member
+olcMemberOfMemberOfAD: memberOf
+```
+
+#### content.ldif
+
+Here is an example of how to configure a running LDAP server, with user `hank` belonging to group `finance`. 
+
+```
+dn: dc=example,dc=com
+changetype: add
+dc: example
+objectClass: dcObject
+objectClass: top
+objectClass: organization
+o: example
+
+dn: ou=users,dc=example,dc=com
+changetype: add
+objectClass: organizationalUnit
+objectClass: top
+ou: users 
+
+dn: ou=groups,dc=example,dc=com
+changetype: add
+objectClass: organizationalUnit
+objectClass: top
+ou: groups
+
+dn: cn=hank,ou=users,dc=example,dc=com
+changetype: add
+objectClass: organizationalPerson
+objectClass: person
+objectClass: top
+sn: hank
+cn: hank
+userPassword: hunter2
+
+dn: cn=finance,ou=groups,dc=example,dc=com
+changetype: add
+objectClass: groupOfNames
+objectClass: top
+member: cn=hank,ou=users,dc=example,dc=com
+cn: finance
+```
+#### Commands
+
+These commands will apply the configuration described in the files above to the LDAP server. 
+
+```
+ldapadd -Y EXTERNAL -H ldapi:/// -f memberOf.ldif
+ldapadd -x -D 'bindDNUser' -w bindDNPassword -H ldapi:/// -f content.ldif
+```
+
+Where bindDNUser and binDNPassword are the bind DN credentials (if configured).
+
+## Application Access Setup 
+
+You need to setup LDAP in the Solace tile correctly, see the [Solace Messaging Documentation]({{ site.links-ldap-settings }}).
+
+In this case, the the username is `hank` and the password is `hunter2`.
+
+Set two environment variables called `LDAP_CLIENTUSERNAME` and `LDAP_CLIENTPASSWORD` with the credentials.
```

Comment:
This is an operating system environment variable you are setting? May want to make that clear... Also wouldnt that be a major security issue by having the password in plain text?